pkg/controller/upload_controller.go

```diff
@@ -294,14 +338,17 @@ func (c *uploadController) processUpload(req *pluginv1api.Upload) error {
 		return errors.New(errMsg)
 	}
 
+	// Unregister on-going upload
+	c.dataMover.UnregisterOngoingUpload(peID)
```

Comment:
would it be more accurate to do the unregister right after upload is completed, compared to do it after the cleanup of local snapshot?